abs.yaml

```diff
@@ -0,0 +1,3 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/cryptography-vectors-feedstock/pr19/961a3d5
+  - https://staging.continuum.io/prefect/fs/setuptools-rust-feedstock/pr5/855001d
```

Comment:
@Jrice1317 You should probably create another PR to remove these from here.

Reply:
Yeah, when the merge build failed, I remembered I forgot about those! Lol thanks

Reply:
This is yet another reason `cryptography-vectors` should be merged with `cryptography`